website/templates/search.html

```diff
@@ -28,19 +28,37 @@ <h1>Result</h1>
6\n2018-09-24,47.00249139809869\n2018-09-25,49.26345096250704\n2018-09-26,53.84601057569515\n2018-09-27,56.00233074653867\n2018-09-28,56.22595050936509\n2018-09-29,56.48197501006925\n2018-09-30,58.556324715096686\n2018-10-01,59.33414600299362\n2018-10-02,55.460655832442725\n2018-10-03,55.518934651923686\n2018-10-04,57.70758580247529\n2018-10-05,58.18741125681837\n2018-10-06,59.6934352783094\n2018-10-07,59.251868261233426\n2018-10-08,62.09214618009842\n2018-10-09,58.656544011430206\n2018-10-10,60.43506757632493\n2018-10-11,60.91669904987496\n2018-10-12,60.87551948434721\n2018-10-13,62.62287417397486\n2018-10-14,62.924858056229965\n2018-10-15,58.06449047270733\n2018-10-16,61.62157004952882\n2018-10-17,64.826417481508\n2018-10-18,62.57460299316195\n2018-10-19,61.797049323679644\n2018-10-20,62.96156627078115\n2018-10-21,66.48387674072579\n2018-10-22,65.91821263149275\n2018-10-23,67.58108069312266\n2018-10-24,66.66318789746265\n2018-10-25,71.30694034798924\n2018-10-26,70.61027458009846\n2018-10-27,70.1090855961651\n2018-10-28,69.73846163025819\n2018-10-29,72.45309581400112\n2018-10-30,71.91070538182053\n2018-10-31,75.62955019637035\n2018-11-01,73.13717021996169\n2018-11-02,72.76696564689209\n2018-11-03,73.86803440226049\n2018-11-04,74.1625162058746\n2018-11-05,75.37755021752808\n2018-11-06,76.00266347607106\n2018-11-07,76.50459700469996\n2018-11-08,76.69465470432291\n2018-11-09,76.20815772230479\n2018-11-10,77.80602061682075\n2018-11-11,78.73529710476042\n2018-11-12,77.5100911847757\n2018-11-13,79.02521044694105\n2018-11-14,81.36472401761517\n2018-11-15,80.74886454908996\n2018-11-16,78.01101349652862\n2018-11-17,78.77996070923048\n2018-11-18,78.24763200296647\n2018-11-19,78.14866376123814\n2018-11-20,78.23693401641174\n2018-11-21,75.9218380109522\n2018-11-22,79.1449984788987\n2018-11-23,80.24500390070267\n2018-11-24,81.13676829572331\n2018-11-25,79.20336704412915\n2018-11-26,81.75938311799982\n2018-11-27,84.1633897152885\n2018-11-28,83.0579205233257\n2018-11-29,83.86192910529121\n2018-11-30,87.19356239436081\n2018-12-01,85.68042795432406\n2018-12-02,81.83931681931566\n2018-12-03,81.48783355213561\n2018-12-04,77.59094883907153\n2018-12-05,74.90041749001415\n2018-12-06,81.71577839746304\n2018-12-07,80.30758264662339\n2018-12-08,80.04023789951044\n2018-12-09,78.67328862182072\n2018-12-10,75.84365828162808\n2018-12-11,77.03900627003091\n2018-12-12,74.81072203377511\n2018-12-13,73.77506739733711\n2018-12-14,72.90836460186027\n2018-12-15,74.81058775552661\n2018-12-16,71.6469596706116\n2018-12-17,68.33858188174187\n2018-12-18,70.0180849418022\n2018-12-19,71.40771636113625\n2018-12-20,70.34278706349627\n2018-12-21,69.73693305309689\n2018-12-22,71.05506349437155\n2018-12-23,66.53593759464832\n2018-12-24,64.47516653363486\n2018-12-25,66.06707059753789\n2018-12-26,65.55444767373119\n2018-12-27,67.81395141997012\n2018-12-28,70.40414406756204\n2018-12-29,71.25919591854708\n2018-12-30,73.03043502137201\n2018-12-31,71.28493548585662\n2019-01-01,73.53905366715313\n';
```

Comment:
What should the default behavior be when a user hasn't clicked on a graph yet? This is fine for now if you're over it, but we have to think about this eventually.